src/swaps/fastbtc/FastbtcSwapProvider.ts

```diff
@@ -80,6 +83,13 @@ class FastbtcSwapProvider extends SwapProvider {
     });
   }
 
+  public getClient(network: Network, walletId: string, asset: string, accountId: string) {
+    return super.getClient(network, walletId, asset, accountId) as Client<
```

Comment:
hmm, isn't this also called to create an RSK client ? Then why would it be bitcoin type only

Reply:
was not aware... will fix that.. however I only did it because of "getTotalFees" which is BTC specific